Update to last SDK, more synth tokens for operators

juanjux opened the pull request:
This commits includes @smola update to the Python driver + more updates by me and the pending PR (closed since this includes it) with all the missing synthetic tokens for operators.

- Synthetic token rules for all operators
- Escaped % synthetictoken do it doesnt intefere with sprintf
- Updated integration tests
- Added missing UnaryOp tokens
- Use errors.new
- Synced with the SDK changes.

smola commented on driver/normalizer/annotation.go:
```diff
@@ -1,7 +1,9 @@
 
 import (
+	"errors"
 	"github.com/bblfsh/python-driver/driver/normalizer/pyast"
```
There should be a space between standard library imports and internal imports.

juanjux replied:
@smola done.

juanjux commented:
Updated to the last SDK changes. Updated the list of missing things, fixed a couple, removed some old comments.

@smola, please review en merge when you can.
